src/Symfony/Component/RateLimiter/RateLimiterFactory.php

```diff
@@ -69,7 +69,11 @@ protected static function configureOptions(OptionsResolver $options): void
     {
         $intervalNormalizer = static function (Options $options, string $interval): \DateInterval {
             try {
-                return (new \DateTimeImmutable())->diff(new \DateTimeImmutable('+'.$interval));
+                // Force UTC timezone, because we don't want to deal with quirks happening when modifying dates in case
+                // there is a default timezone with DST. Read more here https://github.com/symfony/symfony/pull/58757
```

Comment:
ah sorry I missed this one - we don't put references to github in the codebase - instead we put PR discussion in git notes (and git blame can help also)
```suggestion
                // there is a default timezone with DST.
```